streams/src/main/java/org/apache/kafka/streams/state/internals/ThreadCache.java

```diff
@@ -201,6 +201,14 @@ public MemoryLRUCacheBytesIterator reverseAll(final String namespace) {
         return new MemoryLRUCacheBytesIterator(cache.reverseAllKeys(), cache);
     }
 
+    public MemoryLRUCacheBytesIterator prefixScan(final String namespace, final Bytes from, final Bytes to) {
```

Comment:
This method need unit testing. I realized that also here we missed to cover all methods with unit tests. For example, there are also no unit tests for `all()`. Could you please add the ones for `prefixScan()`? Also here you are really welcome to add unit tests for the other methods in a separate PR, but it is not a requirement for the approval of this PR.

Reply:
I created a ticket for adding test cases for the missed methods in ThreadCache. https://issues.apache.org/jira/browse/KAFKA-10767